spec/ruby_terraform/commands/plan_spec.rb

```diff
@@ -12,32 +12,47 @@
   end
 
   it 'calls the terraform plan command passing the supplied directory' do
-    command = RubyTerraform::Commands::Plan.new(binary: 'terraform')
+    command = RubyTerraform::Commands::Plan.new(binary: 'terraform', logger: @logger_mock)
```

Comment:
I typically don't like hiding test state away outside of the test. Would you be able to move the mock creation inline into each test?